d9d03f30b579f1ab-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 's has locations throughout Southern California including San Diego, Los Angeles,' at line 1")
&0bd3dd7d524a422d-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 's Furniture in the  Baton Rouge and Lafayette, Louisiana for all of your furnitu' at line 1")
&itzi4fcf570f225b270ce1701d7593e9dfce-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'arredamento casa e arredamento ufficio. Leader nelle forniture contract di arred' at line 1")
&